bolt/include/bolt/Core/DIEBuilder.h

```diff
@@ -129,6 +129,9 @@ class DIEBuilder {
   uint64_t UnitSize{0};
   llvm::DenseSet<uint64_t> AllProcessed;
   DWARF5AcceleratorTable &DebugNamesTable;
+  // Unordered maps to handle name collision if output DWO directory is
```

Comment:
`maps` -> `map`. Also maybe you can just remove the mention of map as it's implicit in the type.